### DISCUSSION TITLE
Discord-Server

### DISCUSSION THREAD

**Discussion by RGregat** [sceneview/sceneform-android#172] [Ideas]
Anyone here interested to use Discord for any kind of internal communications and maybe customer support for supporter?
I would setup and maintain the Discord-Server. Here is a link to a fresh server without any bots at the moments, but it could be te start :) https://discord.gg/UbNDDBTNqb

**Comment by ThomasGorisse** [comment]
Great initiative but I'm not really convinced about Discord usage as an Issue Tracker.
I have seen the case with [HomeAssistant ](https://www.home-assistant.io/) which I'm using a lot and which manages all its issues within discord and as a result, you cannot find any answer to simple recurrent questions doing a web search.
Also same questions are answered thousand times on the chat because of not classified history search and users preferring asking before searching.

BUT, it's a good idea for having a discussion space between Sceneform users if we always redirect issues/usage questions to GitHub instead of answering there.

**Reply by RGregat** [reply]
I agree to the Issue handling. It makes no sense to do it on Discord. I guess it will be beneficial, when it comes to discuss of writing articles, do videos or what the next steps for the development are.
